modules/nf-core/custom/tx2gene/environment.yml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "custom_tx2gene"
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python=3.9.5
```

Comment:
It's how it's been in RNA-seq and the checksums match with the container, so I'm assuming the minor python differences are not impactful here. I'm inclined to leave well enough alone.